src/main/java/io/cryostat/recordings/RecordingTargetHelper.java

```diff
@@ -136,8 +136,15 @@ public List<IRecordingDescriptor> getRecordings(ConnectionDescriptor connectionD
                 connection -> connection.getService().getAvailableRecordings());
     }
 
+    public enum replacementPolicy {
```

Comment:
To adhere to Java naming policies, the enum name should be TitleCased like `ReplacementPolicy`.